lib/src/base-styles/_nx-text-input.scss

```diff
@@ -5,33 +5,38 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-text-input {
-  border: $nx-border-dark;
-  border-left-width: 4px;
-  border-radius: 0;
-  box-shadow: none;
+.nx-text-input__input {
+  border: 1px solid #d1d6f0;
+  border-radius: $nx-border-radius;
+  color: $nx-text-color-light;
+  font-size: $nx-font-size;
   height: $nx-form-element-height;
-  padding: 0 8px;
+  padding: 0 $nx-spacing-md;
   width: $nx-form-element-width-normal;
 
+  &:hover {
+    border-color: #2c2c2c;
```

Comment:
This doesn't seem to do anything

Reply:
> This doesn't seem to do anything

could you expand upon this a little more?

Reply:
> could you expand upon this a little more?

That comment was more for @glenhunter than for you.  Currently, despite this style, I was never able to get the hover styles to appear.  I believe they were always being overridden by other styles.  Same with the focus style below.

Reply:
> This doesn't seem to do anything

fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/600837d63c526088a976a7de4384b2426cb07531